planning/behavior_velocity_planner/src/scene_module/intersection/util.cpp

```diff
@@ -293,7 +293,7 @@ bool getStopLineIndexFromMap(
   const LineString2d extended_stop_line =
     planning_utils::extendLine(p_start, p_end, planner_data->stop_line_extend_length);
 
-  for (size_t i = lane_interval_start; i <= lane_interval_end; i++) {
+  for (size_t i = lane_interval_start; i <= lane_interval_end - 1; i++) {
```

Comment:
Applying subtraction to an unsigned number is not recommended. It will cause an overflow when the lane_interval_end = 0.

Reply:
Why don't you just use start < end?

Reply:
Yes exactly !